test/integration/targets/dcnm_vrf/tests/nxapi/create.yaml

```diff
@@ -0,0 +1,68 @@
+---
+- debug: msg="START connection={{ ansible_connection }}/create.yaml"
+
+- name: Verify if fabric - vxlan-shrishail-green-field is deployed.
+  dcnm_rest:
+    method: GET
+    path: /rest/control/fabrics/vxlan-shrishail-green-field
+  ignore_errors: yes
+  register: result
+
+- assert:
+    that:
+      - 'result.response|length > 0'
```

Comment:
You probably don't need this assert if you remove `ignore_errors` above.

Reply:
I can take out the ignore_errors, but, still need to keep the assert. The GET API does not return error (400 or 500) when it does not find the fabric.

Reply:
👍